ownCloud Share Extension/ShareViewController.swift

```diff
@@ -66,36 +73,44 @@ class ShareViewController: MoreStaticTableViewController {
 		let bookmarks : [OCBookmark] = OCBookmarkManager.shared.bookmarks as [OCBookmark]
 		if bookmarks.count > 0 {
 			if bookmarks.count > 1 {
-				let rowDescription = StaticTableViewRow(label: "Choose an account and folder to import the file into.".localized, alignment: .center)
+				let rowDescription = StaticTableViewRow(label: "Choose an account and folder to import into.".localized, alignment: .center)
 				actionsRows.append(rowDescription)
 
 				for (bookmark) in bookmarks {
 					let row = StaticTableViewRow(buttonWithAction: { (_ row, _ sender) in
 						self.openDirectoryPicker(for: bookmark, pushViewController: true)
-					}, title: bookmark.shortName, style: .plain, image: Theme.shared.image(for: "owncloud-logo", size: CGSize(width: 25, height: 25)), imageWidth: 25, alignment: .left)
+					}, title: bookmark.shortName, style: .plain, /* make brandable before re-adding --> image: Theme.shared.image(for: "owncloud-logo", size: CGSize(width: 25, height: 25)), imageWidth: 25, */ alignment: .left)
```

Comment:
I will change this directly and using the brandable icon `bookmark-icon.png` instead of.